code/src/java/pcgen/persistence/lst/LstSystemLoader.java

```diff
@@ -61,9 +59,7 @@ public void setChosenCampaignSourcefiles(List<URI> l, GameMode game)
 			StringUtil.join(files, ", "));
 	}
 
-	/**
-	 * @see pcgen.persistence.SystemLoader#getChosenCampaignSourcefiles(pcgen.core.GameMode)
-	 * 
+	/*
```

Comment:
tiny nitpick - don't lose the javadoc syntax: 
```suggestion
	/**
```